### ISSUE TITLE
Remove error for Period < 60 for metrics in the AWS namespace

### ISSUE THREAD

**Issue by Willbly** [aws-cloudformation/cfn-lint#3553]
### CloudFormation Lint Version

1.9.1

### What operating system are you using?

Windows

### Describe the bug

When creating an [AWS::CloudWatch:Alarm](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-cw-alarm.html) with a namespace value prefixed with AWS and a period less than 60 a linting error is produced:

>E3615 30 is less than the minimum of 60

This error was added in https://github.com/aws-cloudformation/cfn-lint/issues/1059 that seems no longer relevant. I have found no AWS documentation describing this behaviour. The AWS page linked above simply says
>The period, in seconds, over which the statistic is applied. This is required for an alarm based on a metric. Valid values are 10, 30, 60, and any multiple of 60.

### Expected behavior

No error should be reported by linting.

### Reproduction template

```
  EnvironmentHealthSevereAlarm:
    Type: AWS::CloudWatch::Alarm
    Properties:
      AlarmActions:
        - !ImportValue Core-CloudWatchAlarmTopicArn
      AlarmName: !Sub "[${EnvironmentName}] Environment Health Severe"
      AlarmDescription: !Sub "The environment health of ${EnvironmentName} is severe"
      ComparisonOperator: GreaterThanOrEqualToThreshold
      EvaluationPeriods: 7
      MetricName: EnvironmentHealth
      Namespace: AWS/ElasticBeanstalk
      Period: 30
      Statistic: Maximum
      Threshold: 25
      TreatMissingData: missing
      Dimensions:
        - Name: EnvironmentName
          Value: !Sub "${EnvironmentName}"
```
